icu4c/source/i18n/unitconverter.cpp

```diff
@@ -5,14 +5,97 @@
 
 #if !UCONFIG_NO_FORMATTING
 
+#include "charstr.h"
+#include "resource.h"
+#include "unicode/stringpiece.h"
+#include "unicode/unistr.h"
+#include "uresimp.h"
+
 U_NAMESPACE_BEGIN
 
+namespace {
+
+//////////////////////////
+/// BEGIN DATA LOADING ///
+//////////////////////////
+
+// Loaded Data Skeleton.
+enum Constants {
+  CONSTANT_FT2M, // ft2m stand for foot to meter.
+  CONSTANT_PI,
+  CONSTANT_G,      // G stands for Gravity.
+  CONSTANT_CUP2M3, // CUP2M3 stand for cup to cubic meter.
+
+  // Must be the last element.
+  CONSTANTS_COUNT
+};
+
+struct Factor {
+  int64_t factorNum;
```

Comment:
Why is your formatter inserting 2 spaces?  It should insert 4 spaces.

Reply:
I have just fixed it.